src/molecules/FullMolecule.h

```diff
@@ -261,8 +261,33 @@ class FullMolecule : public MoleculeInterface {
 		}
 		return d2;
 	}
-	
+	//calculates orientation angle for ARDF
+	double orientationAngle(const FullMolecule& molecule2, double dr[3], double d2) const {
```

Comment:
I suppose that this function should also find its way into the MoleculeInterface, as otherwise both the RMM, as well as the AutoPas mode will be broken.
So please add this to the interface :)

Reply:
If I move it to MoleculeInterface.h, would I not use const MoleculeInterface& molecule2?

Reply:
Also do I need to have the function in both FullMolecule and MoleculeInterface or only in the latter? Because I see the function dist2 can be found in both.

Reply:
MoleculeInterface -> yes that is true.
MoleculeInterface only should work. I think dist2 is only there as a possibly more performant variant?